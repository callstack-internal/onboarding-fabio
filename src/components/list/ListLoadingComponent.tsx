import React from 'react';
import {ActivityIndicator, StyleSheet} from 'react-native';
import {SafeAreaView} from 'react-native-safe-area-context';
import Colors from '../../styles/Colors';

const ListLoadingComponent = () => {
  return (
    <SafeAreaView style={styles.rootContainer} edges={['left', 'right']}>
      <ActivityIndicator size={'large'} color={Colors.black} />
    </SafeAreaView>
  );
};

const styles = StyleSheet.create({
  rootContainer: {
    alignItems: 'center',
    paddingHorizontal: 20,
    paddingVertical: 40,
  },
});

export default ListLoadingComponent;
