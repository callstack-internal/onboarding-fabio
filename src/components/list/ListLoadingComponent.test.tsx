import React from 'react';
import renderer from 'react-test-renderer';
import ListLoadingComponent from './ListLoadingComponent';

test('should render correctly', () => {
  const tree = renderer.create(<ListLoadingComponent />).toJSON();

  expect(tree).toMatchSnapshot();
});
