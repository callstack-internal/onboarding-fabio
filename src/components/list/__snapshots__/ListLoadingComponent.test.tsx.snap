// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<RNCSafeAreaView
  edges={
    [
      "left",
      "right",
    ]
  }
  style={
    {
      "alignItems": "center",
      "paddingHorizontal": 20,
      "paddingVertical": 40,
    }
  }
>
  <ActivityIndicator
    color="#000000"
    size="large"
  />
</RNCSafeAreaView>
`;
